src/Resources/views/storefront/page/search/index.html.twig

```diff
@@ -34,6 +34,11 @@
                             '%filterValue%': queryInfoMessage.filterValue,
                             '%hits%': hits
                         })|sw_sanitize }}
+                    {% elseif queryInfoMessage is instanceof("\\FINDOLOGIC\\FinSearch\\Struct\\QueryInfoMessage\\ShoppingGuideInfoMessage") %}
```

Comment:
Please add this as the first check, to keep the code in the response parser in sync with the view.